src/rpc/rawtransaction.cpp

```diff
@@ -149,6 +149,7 @@ static std::vector<RPCArg> CreateTxDoc()
         },
         {"outputs", RPCArg::Type::ARR, RPCArg::Optional::NO, "The outputs (key-value pairs), where none of the keys are duplicated.\n"
                 "That is, each address can only appear once and there can only be one 'data' object.\n"
+                "Either {\"data\"} or {\"address\"}, are required, but specifying both is optional.\n"
```

Comment:
How about something along these lines?

```suggestion
        {"outputs", RPCArg::Type::ARR, RPCArg::Optional::NO, "The transaction’s outputs specified as key-value pairs.\n"
                 "Payment outputs are specified in the form of \"address\":\"amount [BTC]\", e.g. \"bc1p…xyz\":\"0.001\".\n"
                 "A single data output may be specified in the form of \"data\":\"hex\", e.g. \"data\":\"ff00ff00ff\".\n"
                 "Each key may only appear once. This means that each address may only appear once, and there can only be one 'data' output.\n"
                 "At least one output must be specified.\n"
```